lib/src/library.dart

```diff
@@ -70,6 +72,77 @@ class LibraryReader {
     }
   }
 
+  /// Returns a [Uri] from the current library to the one provided.
+  ///
+  /// If possible, a `package:` or `dart:` URL scheme will be used to reference
+  /// the library, falling back to relative paths if required (such as in the
+  /// `test` directory).
+  ///
+  /// May throw [ArgumentError] if it is not possible to resolve a path.
```

Comment:
This should mention the schemes it supports

Reply:
Done! Thanks.